app/routers/guest/applications.py

```diff
@@ -464,7 +465,9 @@ async def update_apps_send_notifications(
     session: Session = Depends(get_db),
     client: CognitoClient = Depends(dependencies.get_cognito_client),
     application: models.Application = Depends(
-        dependencies.get_scoped_publication_as_guest_via_payload(statuses=(models.ApplicationStatus.ACCEPTED,))
+        dependencies.get_scoped_publication_as_guest_via_payload(
+            statuses=(models.ApplicationStatus.ACCEPTED,), scopes=(ApplicationScope.UNEXPIRED,)
```

Comment:
I put statuses last for consistency.

```suggestion
            scopes=(ApplicationScope.UNEXPIRED,), statuses=(models.ApplicationStatus.ACCEPTED,)
```